tests/test_functons.py

```diff
@@ -12,56 +12,88 @@
 
 
 @pytest.mark.parametrize("test_square_input,test_square_expected", [
+    (-(3037000499**2-1), -sys.maxsize-1),
+    (-(3037000499**2), -sys.maxsize),
```

Comment:
actually it was my fault - we don't need extra parentheses here, so you can just
`-3037000499**2-1`
Ok, i think you accidentally switched input/expected output... We partition input values into equivalent partitions. So the boundaries for those partitions are `-sys.maxsize-1`, `0`, `sys.maxsize`
so you have to apply boundary condition analysis to those boundaries (x-1, x, x+1) and pick one values from the each partition: `-sys.maxsize**2`, `-sys.maxsize**0.5` (power of 0.5 == square root), `sys.maxsize**0.5`, and `sys.maxsize**2`.